File: app/src/main/java/com/cottondroid/todoachieved/task/form/time/DateTextView.java
package com.cottondroid.todoachieved.task.form.time;


import android.content.Context;
import android.util.AttributeSet;

import com.cottondroid.todoachieved.R;

import java.text.DateFormat;

public class DateTextView extends DateTimeTextView {

    public DateTextView(Context context) {
        super(context);
    }

    public DateTextView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public DateTextView(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    @Override
    public DateFormat getDateFormat() {
        return DateFormat.getDateInstance();
    }

    @Override
    public int getDefaultText() {
        return R.string.task_form_choose_date;
    }
}

File: app/src/main/java/com/cottondroid/todoachieved/task/form/time/DateTextView.kt
package com.cottondroid.todoachieved.task.form.time

import android.content.Context
import android.util.AttributeSet
import com.cottondroid.todoachieved.R
import java.text.DateFormat

class DateTextView(context: Context, attrs: AttributeSet) : DateTimeTextView(context, attrs) {
    override val dateFormat: DateFormat = DateFormat.getDateInstance()
    override val defaultText: Int = R.string.task_form_choose_date
}